builder/vmware/common/driver.go

```diff
@@ -642,8 +647,46 @@ func GetOVFTool() string {
 	return ovftool
 }
 
+func CheckOvfToolVersion(ovftoolPath string) bool {
+	output, err := exec.Command(ovftoolPath, "--version").CombinedOutput()
+	if err != nil {
+		return false
```

Comment:
Might be a good idea to log the error/output of the command here

Reply:
Resolved in 939a252.